tools/visualize-benchmark-stats.sh

```diff
@@ -23,13 +28,14 @@ data_file="${1}"
         set output 'benchmark-stats-pipelineruns.png'
         plot"
     echo -n " '$data_file' using 2:4 title 'prs_total' linewidth 2,"
-    echo -n " '$data_file' using 2:5 title 'prs_pending' linewidth 2,"
-    echo -n " '$data_file' using 2:6 title 'prs_running' linewidth 2,"
-    echo -n " '$data_file' using 2:7 title 'prs_finished' linewidth 2,"
-    echo -n " '$data_file' using 2:8 title 'prs_signed_true' linewidth 2,"
-    echo -n " '$data_file' using 2:10 title 'prs_finalizers_present' linewidth 2,"
-    echo -n " '$data_file' using 2:14 title 'prs_deleted' linewidth 2,"
-    echo -n " '$data_file' using 2:15 title 'prs_terminated' linewidth 2,"
+    echo -n " '$data_file' using 2:5 title 'prs_failed' linewidth 2,"
+    echo -n " '$data_file' using 2:6 title 'prs_pending' linewidth 2,"
+    echo -n " '$data_file' using 2:7 title 'prs_running' linewidth 2,"
+    echo -n " '$data_file' using 2:8 title 'prs_finished' linewidth 2,"
+    echo -n " '$data_file' using 2:9 title 'prs_signed_true' linewidth 2,"
+    echo -n " '$data_file' using 2:11 title 'prs_finalizers_present' linewidth 2,"
+    echo -n " '$data_file' using 2:15 title 'prs_deleted' linewidth 2,"
+    echo -n " '$data_file' using 2:16 title 'prs_terminated' linewidth 2,"
 } | gnuplot
```

Comment:
At some point, we maybe want to use your yaml-configured tool to lot these graphs

Reply:
Yep, that would be the long term plan. I'll start working on that utility at the side.